app/controllers/waste_exemptions_engine/last_email_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class LastEmailController < ApplicationController
+    def show
+      if ENV["ENABLE_LAST_EMAIL_CACHE"] == "true"
+        render json: WasteExemptionsEngine::LastEmailCacheService.instance.last_email_json
+      else
+        handle_disabled
```

Comment:
`handle_disabled` is a one line method. As such I think we can just include that one line here, but there may be a case or reason I am not seeing.

Reply:
Fixed in 15b3b13.